### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by sampaiogil** [google/or-tools#3478]
Hi,

I'm trying to compile a program in Ubuntu (20.04) using VS2019 in Windows via WSL but I'm getting the following error message:
_/usr/include/ortools/linear_solver/linear_solver.h(149,10): error : absl/base/port.h: No such file or directory_

I've followed the official guide (https://developers.google.com/optimization/install/cpp/source_linux) and its seems the examples run just fine.

Am I doing something wrong?

Thanks!

**Comment by Mizux** [comment]
what do you means by "a program" a custom one ? then can you share the CMakeLists.txt you did use ? the command line used ? any releavant trace ?

seems we miss a dependenciy on absl/base/port.h

**Comment by sampaiogil** [comment]
Yes, it's a project I developed first in Windows and need to compile it now for linux. 

I just followed the guide:
sudo apt-get -y install git wget pkg-config build-essential cmake autoconf libtool zlib1g-dev lsb-release
git clone https://github.com/google/or-tools
make third_party
make cc
make test_cc
make cpp

The ortools directory was actually created in /usr/include/ but no sign of the absl directory...

Thank you for your help!

**Comment by Mizux** [comment]
take a look at `install_make`....
